Tasks/PublishPipelineArtifactV1/task.loc.json

```diff
@@ -0,0 +1,47 @@
+{
+  "id": "ECDC45F6-832D-4AD9-B52B-EE49E94659BE",
+  "name": "PublishPipelineArtifact",
+  "friendlyName": "ms-resource:loc.friendlyName",
+  "description": "ms-resource:loc.description",
+  "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/publish-pipeline-artifact",
+  "helpMarkDown": "ms-resource:loc.helpMarkDown",
+  "category": "Utility",
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 1,
+    "Minor": 0,
+    "Patch": 0
+  },
+  "groups": [],
+  "demands": [],
+  "preview": true,
+  "minimumAgentVersion": "2.140.1",
+  "inputs": [
+    {
+      "name": "path",
```

Comment:
Are there requirements to change the order from V0? artifactName <-> targetPath(path) is reversed? If not, I feel we should not change them.

Reply:
Since right now the artifact name is optional so we re-ordered it so the targetPath (path) is the first item.